WcaOnRails/app/webpacker/components/StaticPages/Disclaimer/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
Comment about the filename (not sure where to put this, so I'm arbitrarily picking line 1)

If it's only a single component that has no nested components or shared logic, you don't have to create an entire new folder. `.../components/StaticPages/Disclaimer.js` will do.

Reply:
Ok yes, so I was just thinking that some of the static pages would have more than one file, so for the sake of sameness put all of the pages in their own folder but that's fine.